src/tray/mod.rs

```diff
@@ -0,0 +1,132 @@
+mod command;
+mod error;
+
+pub use command::Command;
+pub use error::Error;
+
+use anyhow::Context;
+use relm4::Sender;
+use std::{fs::File, io::Cursor};
+use tracing::{error, info};
+
+use image::{ImageBuffer, Rgba};
+use ksni;
+
+use crate::get_xdg;
+
+const LOCK_FILE: &str = "tray.lock";
```

Comment:
In fact the whole application is running no? If so then maybe

```suggestion
const LOCK_FILE: &str = "flakeshot.lock";
```

Reply:
btw what happens if `flakeshot tray`is running and I run `flakeshot gui`?